packages/camera/camera_android_camerax/CONTRIBUTING.md

```diff
@@ -0,0 +1,75 @@
+`InstanceManager`, which is essentially a map between `long` identifiers and
+objects that also provides notifications when an object has become unreachable
+by memory. There is both a Dart and Java `InstanceManager` implementation, so
+when a Dart instance is created that represens an Android native instance,
+both are stored in the `InstanceManager` of their respective language with a
+shared `long` identifier. These `InstanceManager`s take callbacks that run
+when objects become unrechable or removed, allowing the Dart library to easily
+handle removing references to native resources automatically. To ensure all
+created instances are properly managed and to more easily allow for testing,
+each wrapped Android native class in Dart takes an `InstanceManager` and has
+a detached constructor, a constructor that allows for the creation of instances
+not attached to the `InstanceManager` and unlinked to a paired Android native
+instance.
+
+In `lib/src/`, you will find all of the Dart-wrapped native Android classes that
+the plugin currently uses in its implementation. As aforementioned, each of
+these classes uses an `InstanceManager` (implementation in `instance_manager.dart`)
+to manage objects that are created by the plugin implementation that map to objects
+of the same type created in the native Android code. This plugin uses [`pigeon`][1]
+to generate the communication layer between Flutter and native Android code, so each
+of these Dart-wrapped classes may also have Host API and Flutter API implementations
+that handle communication to the host native Android platform and from the host
+native Android platform, respectively. The communication interface is defined in
+the `pigeons/camerax_library.dart` file. After editing the communication interface,
+regenerate the communication layer by running
+`dart run pigeon --input pigeons/camerax_library.dart` from the plugin root.
+
+In the native Java Android code in `android/src/main/java/io/flutter/plugins/camerax/`,
+you'll find the Host API and Flutter API implementations of the same classes
+wrapped with Dart in `lib/src/` that handle communication from that Dart code
+and to that Dart code, respectively. The Host API implementations should directly
+delegate calls to the CameraX or other wrapped Android libraries and should not
+have any additional logic or abstraction; any exceptions should be thoroughly
+documented in the code. As aforementioned, the objects created in the native
+Android code map to objects created on the Dart side and are also managed by
+an `InstanceManager` (implementation in `InstanceManager.java`).
+
+If CameraX or other Android classes that you need to access do not have a
+duplicately named implementation in `lib/src/`, then follow the same structure
+described above to add them.
+
+For more information on the approach of wrapping native libraries for plugins,
+please see the [design document][2] or contact [@bparrishmines][6] on any
```

Comment:
@bparrishMines is mentioning you like this okay with you?

Reply:
We don't typically include references to our github accounts in the plugin. I think having https://github.com/flutter/packages/blob/main/CODEOWNERS should suffice. It also makes it easier to maintain since I will be added to all camera plugin PRs anyways.

Alternatively you could link to the ecosystem discord channel instead. Then anyone who is able to answer a question will be notified.

Reply:
I like both of those suggestions. In general it is a good idea not try not to encourage contacting a person. It is not great for the person or the one reaching out. (imagine someone 12h timezone offset from @bparrishMines 3 years from now)